bentoml/catboost.py

```diff
@@ -244,24 +251,27 @@ def num_concurrency_per_replica(self) -> int:
     def num_replica(self) -> int:
         return int(round(self.resource_quota.cpu))
 
-    # pylint: disable=arguments-differ,attribute-defined-outside-init
-    def _setup(self) -> None:  # type: ignore[override]
-        self._model = _load_helper(self._model_file, self._model_params)  # type: ignore[var-annotated] # noqa: LN001 # pylint: disable
+    # pylint: disable=attribute-defined-outside-init
+    def _setup(self) -> None:
+        self._model = _load_helper(self._model_file, self._model_params)
         self._predict_fn = getattr(self._model, self._predict_fn_name)
 
-    # pylint: disable=arguments-differ
-    def _run_batch(  # type: ignore[override]
-        self, input_data: t.Union[np.ndarray, "pd.DataFrame", cbt.Pool]
-    ) -> np.ndarray:
-        # Take a batch type
-        # Take a containers
-        res = self._predict_fn(input_data)
+    def _run_batch(
```

Comment:
catboost's `predcit` method only accept one argument as input, there's no need for adding *args and **kwargs in this case

Reply:
see https://github.com/catboost/catboost/blob/master/catboost/python-package/catboost/core.py#L4767